migrator/migrations/frozenschema/v75/notification_schedules.go

```diff
@@ -0,0 +1,33 @@
+package schema
```

Comment:
Should rather be v4.0 instead of v75, no?
Or are we just "counting" up and not associating this with the release?

Reply:
I tried to push for decorrelation from datamodel version from release version in the past, in order to avoid that kind of confusion (see https://github.com/stackrox/stackrox/pull/4839 ).
In the end the feedback I got was (I think it is the point of the next comment):
> each migration (data manipulation or data model definition) should embed the files it needs.

I moved the files in https://github.com/stackrox/stackrox/pull/4968 and here to have the schema part of the migration itself.

Reply:
I think this continues to evolve.  Personally I prefer isolating it to the migration itself because it is easier for me to understand and pinpoint exactly when a schema changed.